README.md

```diff
@@ -0,0 +1,41 @@
+# Chingu
+
+OpenGL accelerated 2D game framework for Ruby.
+Builds on the awesome [Gosu](https://github.com/gosu/gosu) (Ruby/C++) which provides all the core functionality.
+It adds simple yet powerful game states, pretty input handling, deployment safe asset-handling, a basic re-usable game object and automations of common task.
+
+[Old readme](README.old.rdoc)
+
+## Fork
+This is a fork of the Chingu game framework, its purpose is to maintain it up
+to-date with Gosu.
+
+You can find the original repo of Chingu [here](https://github.com/ippa/chingu)
```

Comment:
One thing that would be important to me is that this fork of Chingu stays compatible with the existing Chingu library, even where some parts might be outdated by now. There are many more Gosu games on GitHub and the Gosu Showcase board than in development right now, and it would be good to preserve them. (Same as for Gosu itself.) Maybe this goal is worth pointing out here if you share it :)

Reply:
I see, I'll make sure to mention